core/bentley/package.json

```diff
@@ -52,6 +52,9 @@
     "plugins": [
       "@bentley"
     ],
-    "extends": "plugin:@bentley/imodeljs-recommended"
+    "extends": "plugin:@bentley/imodeljs-recommended",
+    "rules": {
+      "@typescript-eslint/explicit-module-boundary-types": "off"
```

Comment:
I fixed it in packages that were fairly easy to fix and disabled in others to avoid having them pile up before I'm ready to enable it everywhere. This way at least we won't get any new code written against the rule in most packages.

Reply:
Sounds good.  As I mentioned above, lets make sure to let everyone know that they should go through and fix it.